Library/Formula/dcd.rb

```diff
@@ -0,0 +1,37 @@
+class Dcd < Formula
+  homepage "https://github.com/Hackerpilot/DCD"
+  url "https://github.com/Hackerpilot/DCD.git",
+      :tag => "v0.5.1",
+      :revision => "351bf2ee2d5f1c4986c2c5957f542dda17b1d085"
+
```

Comment:
Can you add this at the following line:
```
bottle do
end
```

Sorry about the need for this, it's just because of how our binary packages get uploaded and it's getting grumpy in this case.